handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -1182,7 +1254,26 @@ static KeyManagerFactory buildKeyManagerFactory(X509Certificate[] certChain, Pri
                                                     KeyManagerFactory kmf)
             throws UnrecoverableKeyException, KeyStoreException, NoSuchAlgorithmException,
             CertificateException, IOException {
-        return buildKeyManagerFactory(certChain, KeyManagerFactory.getDefaultAlgorithm(), key, keyPassword, kmf);
+        return buildKeyManagerFactory(certChain, key, keyPassword, kmf, KeyStore.getDefaultType());
+    }
+
+    static KeyManagerFactory buildKeyManagerFactory(X509Certificate[] certChain, PrivateKey key, String keyPassword,
```

Comment:
Ditto. Modify existing versions instead of creating a new one. Unless there's a lot of callers.

Ditto for next function.

Reply:
There are 6 methods. I am not sure if I should delete the method or not

Reply:
> There are 6 methods. I am not sure if I should delete the method or not

I just deleted it as I can simply add a null parameter for each usage.

Reply:
As long as these are all package-private it is fine to remove... thanks!